include/vast/Translation/CodeGenUnsupportedVisitor.hpp

```diff
@@ -0,0 +1,527 @@
+// Copyright (c) 2023-present, Trail of Bits, Inc.
+
+#pragma once
+
+#include "vast/Util/Warnings.hpp"
+#include "vast/Translation/CodeGen.hpp"
+#include "vast/Translation/CodeGenContext.hpp"
+#include "vast/Translation/CodeGenVisitor.hpp"
+#include "vast/Translation/CodeGenBuilder.hpp"
+#include "vast/Translation/CodeGenFallBackVisitor.hpp"
+
+namespace vast::cg
+{
+
+#define VISITOR_DECL(decl_type) \
```

Comment:
Unscoped name that is not properly unique. Also undefine at the end.

Reply:
undef the macro at the end to avoid conflict with other macros with same name.